### PR TITLE
Show forbidden notification if there is no read permission

### PR DESCRIPTION
**Pull request by fscholdei**
## Proposed changes

Show forbidden notification if there is no read permission

### Your checklist for this pull request

- [x] PR is well described and the description can be used as commit message on squash
- [x] Related issues linked to PR if existing and labels set
- [x] Target branch is not master (in most cases develop should bet the target of choice) 
- [x] Code does not conflict with target branch
- [ ] New code is covered with unit tests
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog`
- [x] Definition of Done's fulfilled: [DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/definition-of-done.md) // [UI DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/ui-dod.md)
- [ ] Documentation updated (only necessary for new features or changed behaviour)

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: src/main/js/SinglePullRequest.tsx [hunk 1]
```diff
@@ -22,18 +22,20 @@
...
 
-type Props = RouteComponentProps & {
-  repository: Repository;
-  userAutocompleteLink: string;
-};
+type Props = RouteComponentProps &
+  WithTranslation & {
+    repository: Repository;
+    userAutocompleteLink: string;
+  };
```
Comment (eheimbuch) [inline src/main/js/SinglePullRequest.tsx:38]: You migrated one component to FC but not the others? I don't get it.

File: src/main/resources/locales/en/plugins.json [hunk 2]
```diff
@@ -137,6 +137,7 @@
     "pullRequests": {
       "errorTitle": "Error",
       "errorSubtitle": "Unspecified pull request error",
+      "forbidden": "Please note: You do not have the permission to read pull requests!",
```
Comment (eheimbuch) [inline src/main/resources/locales/en/plugins.json:140]:
```suggestion
      "forbidden": "You do not have the permission to read pull requests!",
```

File: src/main/resources/locales/de/plugins.json [hunk 3]
```diff
@@ -137,6 +137,7 @@
     "pullRequests": {
       "errorTitle": "Fehler",
       "errorSubtitle": "Unbekannter Pull Request Fehler",
+      "forbidden": "Hinweis: Es fehlen Berechtigungen zum Lesen der Pull Requests!",
```
Comment (eheimbuch) [inline src/main/resources/locales/de/plugins.json:140]:
```suggestion
      "forbidden": "Es fehlen Berechtigungen zum Lesen der Pull Requests!",
```

**Review by eheimbuch — CHANGES_REQUESTED** [review]

File: src/main/js/SinglePullRequest.tsx (hunk 1 above)
Comment (fscholdei) [reply src/main/js/SinglePullRequest.tsx:38]: Well, good point 😵

**Pull request merged**